nexus/ocp-config/dc.yml

```diff
@@ -26,7 +45,22 @@ objects:
           app: nexus3
           deploymentconfig: nexus3
       spec:
+        serviceAccount: nexus3
+        serviceAccountName: nexus3
         containers:
+        - name: oauth-proxy
+          image: openshift/oauth-proxy:latest
+          imagePullPolicy: IfNotPresent
+          ports:
+          - containerPort: 8082
+            name: public
+          args:
+          - --https-address=
```

Comment:
this stops oauth-proxy from loading tls stuff.
in this pr ssl termination is only done in openshift